ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -123,4 +141,19 @@ export class VaultMaxiProgram extends CommonProgram {
         return true
     }
 
+    async cleanUp(vault: LoanVaultActive, telegram: Telegram): Promise<boolean> {
+        const tokens = await this.getTokenBalances()
+        let paybackTokens: TokenBalance[] = []
+        let tokenIdToSymbol = new Map<Number, string>()
+
+        vault.loanAmounts.forEach(loan => {
+            let token = tokens.get(loan.symbol)
+            if (token) {
+                tokenIdToSymbol.set(+token.id, token.symbol)
+                paybackTokens.push({ token: +token.id, amount: new BigNumber(+token.amount) })
```

Comment:
maybe we should change `token.amount` to `Math.min(+token.amount, loan.amount)`

Reply:
no, overpay does't matter for the repay,